torch/_refs/__init__.py

```diff
@@ -3134,15 +3134,17 @@ def native_group_norm(
         [batch_size, num_groups, num_channels // num_groups, flattened_inner_size],
     )
     out, mean, rstd = _normalize(input_reshaped, reduction_dims, eps)
-    out = out.view(input.shape)
 
-    broadcast_dims = [0] + list(range(2, input.ndim))
     unsqueeze_bias = None
     if bias is not None:
-        unsqueeze_bias = _unsqueeze_multiple(bias, broadcast_dims)
```

Comment:
nit:
```
broadcast_dims = [0] + list(range(3, input_reshaped.ndim))
...
unsqueeze_bias = _unsqueeze_multiple(bias.reshape([num_groups, num_channels // num_groups]), broadcast_dims)
```

Reply:
Thanks for your comment, actually these two lines of code are identical to the current implementation:
```
unsqueeze_bias = torch.reshape(
            bias, [1, num_groups, num_channels // num_groups, 1]
        )
```